backends/p4tools/testgen/targets/ebpf/table_stepper.h

```diff
@@ -0,0 +1,57 @@
+#ifndef TESTGEN_TARGETS_EBPF_TABLE_STEPPER_H_
+#define TESTGEN_TARGETS_EBPF_TABLE_STEPPER_H_
+
+#include <map>
+#include <vector>
+
+#include "ir/ir.h"
+#include "lib/cstring.h"
+
+#include "backends/p4tools/testgen/core/small_step/table_stepper.h"
+#include "backends/p4tools/testgen/lib/execution_state.h"
+#include "backends/p4tools/testgen/lib/test_spec.h"
+#include "backends/p4tools/testgen/targets/ebpf/expr_stepper.h"
+#include "backends/p4tools/testgen/targets/ebpf/test_spec.h"
+
+namespace P4Tools {
+
+namespace P4Testgen {
+
+namespace EBPF {
+
+class EBPFTableStepper : public TableStepper {
+ private:
+    /// Specifies the type of the table implementation:
```

Comment:
I don't understand what is the relationship between these enums and ebpf.
Ebpf has only 2 implementation properties, array or hash.

Reply:
Yes, eBPF does not have a lot of additional table properties or implementations. We could add hash and array to this enum, but they are not changing the semantics of entry generation for tests if I recall correctly.